infra/gp-hub-logging/templates/basicAuth.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: loki-basic-auth-secret
+type: Opaque
+data:
+  USER_NAME: {{ default "" .Values.loki.gateway.basicAuth.username | b64enc | quote }}
```

Comment:
braucht man hier den default wert? würde sonst stattdessen lieber in den values einen default wert à la "change-from-inventory" oder sowas machen, wenn du sonst einen fehler beim base64 encoden bekommst.